src/operation/in_memory_model_store.jl

```diff
@@ -282,3 +179,7 @@ function get_parameter_value(
 ) where {T <: ParameterType, U <: Union{PSY.Component, PSY.System}}
     return store.data.parameters[ParameterKey(T, U)]
 end
+
+# TODO DT: Jose, do we need this for DecisionModel?
```

Comment:
Note this question

Reply:
No, last recorded row is not needed for DecisionModel. In the Emulation model, it is useful because we pre-allocate the DataFrame.